query-engine/metrics/src/lib.rs

```diff
@@ -79,11 +79,18 @@ const ACCEPT_LIST: &[&str] = &[
     MOBC_POOL_CONNECTIONS_IDLE,
     MOBC_POOL_WAIT_COUNT,
     MOBC_POOL_WAIT_DURATION,
-    PRISMA_CLIENT_QUERIES_DURATION_HISTOGRAM_MS,
     PRISMA_CLIENT_QUERIES_TOTAL,
-    PRISMA_DATASOURCE_QUERIES_DURATION_HISTOGRAM_MS,
     PRISMA_DATASOURCE_QUERIES_TOTAL,
+    PRISMA_POOL_CONNECTIONS_OPENED_TOTAL,
+    PRISMA_POOL_CONNECTIONS_CLOSED_TOTAL,
+    PRISMA_POOL_CONNECTIONS_OPEN,
+    PRISMA_POOL_CONNECTIONS_BUSY,
+    PRISMA_POOL_CONNECTIONS_IDLE,
```

Comment:
Is it correct to add our "aliases" here?
(Might also apply to the 2 `wait` metrics - not sure)

Reply:
When `describe_metrics()` is called, it emits the initial value of the metrics using the alias. However, the [registry subscriber](https://github.com/prisma/prisma-engines/blob/b52b336a9238cdecd9b62b305cc926a11ad1b72f/query-engine/metrics/src/registry.rs#L304C44-L304C44) will ignore them, because they aren't on the list of accepted metrics. So, currently, those metrics are only registered when mobc emits them.

Reply:
And I assume that has negative impacts, like them being missing in some initial output or similar?
(I know only half the words you used in your comment - I _should_ ask an Engineer to take a look here 😝)

Reply:
Sorry for not being as clear as I'd like 😬. You're right: some metrics may be missing from the first few API outputs. For example, `prisma_pool_connections_closed_total` will only be present in the output after a connection is closed because it's the moment when mobc emits a value for it (using the name `mobc_pool_connections_closed_total`).

Reply:
Thanks. That seems pretty inelegant in our code then. We should revisit this.

Reply:
Really good catch in here, @alencardc Thank you! This part of the codebase needs good shaking!